autogpt_platform/backend/linter.py

```diff
@@ -21,7 +22,7 @@ def lint():
 
 
 def format():
-    run("ruff", "check", "--fix", ".")
-    run("isort", "--profile", "black", ".")
-    run("black", ".")
-    run("pyright", ".")
+    run("ruff", "check", "--fix", *target_dirs)
+    run("isort", "--profile", "black", *target_dirs)
+    run("black", *target_dirs)
+    run("pyright", *target_dirs)
```

Comment:
1. Does this depend on the CWD from where the script is run?
2. This clashes with the linting I set up for `autogpt_libs` in #8584, which only uses `ruff` (and `pyright` but that's not a linter)